common/containers/Tabs/GenerateWallet/components/FinalSteps.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import translate from 'translations';
+import { WalletType } from '../GenerateWallet';
+import SiteImage from 'assets/images/unlock-guide/site.png';
+import TabImage from 'assets/images/unlock-guide/tab.png';
+import SelectKeystoreImage from 'assets/images/unlock-guide/select-keystore.png';
+import ProvideKeystoreImage from 'assets/images/unlock-guide/provide-keystore.png';
+import SelectMnemonicImage from 'assets/images/unlock-guide/select-mnemonic.png';
+import ProvideMnemonicImage from 'assets/images/unlock-guide/provide-mnemonic.png';
+import './FinalSteps.scss';
+
+interface Props {
+  walletType: WalletType;
+}
+
+export default class FinalSteps extends React.Component<Props> {
```

Comment:
This could just be an SFC instead, and then you can have all the logic in a separate function too